features/raadpleeg-verblijfplaats-met-periode/in-onderzoek.feature

```diff
@@ -0,0 +1,622 @@
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde              |
+      | type                | RaadpleegMetPeriode |
+      | burgerservicenummer | 000000012           |
+      | datumVan            | 2023-07-01          |
+      | datumTot            | 2024-01-01          |
+      Dan heeft de response verblijfplaatsen met de volgende gegevens
+      | type  | datumVan.type | datumVan.datum | datumVan.langFormaat | datumTot.type | datumTot.datum | datumTot.langFormaat | gemeenteVanInschrijving.code | gemeenteVanInschrijving.omschrijving |
+      | Adres | Datum         | 2023-10-14     | 14 oktober 2023      | Datum         | 2024-05-16     | 16 mei 2024          | 0800                         | Hoogeloon, Hapert en Casteren        |
+      En heeft de verblijfplaats de volgende 'inOnderzoek' gegevens
+      | naam                             | waarde      |
+      | datumTot                         | true        |
+      | datumIngangOnderzoek.type        | Datum       |
+      | datumIngangOnderzoek.datum       | 2024-05-16  |
+      | datumIngangOnderzoek.langFormaat | 16 mei 2024 |
+      En heeft de verblijfplaats de volgende 'verblijfadres' gegevens
+      | naam                                         | waarde      |
+      | korteStraatnaam                              | Testpad     |
+      | huisnummer                                   | 8           |
+      En heeft de verblijfplaats de volgende 'adressering' gegevens
+      | naam                                         | waarde                        |
+      | adresregel1                                  | Testpad 8                     |
+      | adresregel2                                  | HOOGELOON, HAPERT EN CASTEREN |
+
+      Voorbeelden:
+      | omschrijving                                  | aanduiding onderzoek |
+      | hele categorie verblijfplaats                 | 80000                |
+      | historische categorie verblijfplaats          | 58000                |
+      | hele groep adreshouding                       | 81000                |
+      | historische groep adreshouding                | 581000               |
+      | datum aanvang adreshouding                    | 81030                |
+      | historische datum aanvang adreshouding        | 581030               |
+      | hele groep verblijf buitenland                | 81300                |
+      | hele historische groep verblijf buitenland    | 581300               |
+      | datum aanvang verblijf buitenland             | 81320                |
+      | historische datum aanvang verblijf buitenland | 581320               |
+
+    Abstract Scenario: datumTot staat niet in onderzoek wanneer het onderzoek op de volgende verblijfplaats beëindigd is
+      Gegeven adres 'A1' heeft de volgende gegevens
```

Comment:
Kan dit voorkomen? Bij de data service hebben we gespecificeerd dat beëindigd onderzoek niet wordt geleverd. Zie: https://github.com/BRP-API/historie-data-service/blob/main/features/raadpleeg-verblijfplaats-met-periode/in-onderzoek.feature regel 105 en 122